src/main/java/com/pikachu/purple/infrastructure/persistence/favorite/entity/favoriteJpaEntity.java

```diff
@@ -0,0 +1,35 @@
+package com.pikachu.purple.infrastructure.persistence.favorite.entity;
+
+import com.pikachu.purple.infrastructure.persistence.common.BaseEntity;
+import com.pikachu.purple.infrastructure.persistence.common.Status;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Entity
+@Table(name = "favorite")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class favoriteJpaEntity extends BaseEntity {
+
+    @Id
+    @Column(name = "favorite_id")
+    private Long favoriteId;
+
+    @Column(name = "user_id")
+    private Long userId;
+
+    @Column(name = "perfume_id")
+    private Long perfumeId;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "favorite_status")
+    private Status favoriteStatus;
```

Comment:
p3) 이렇게 가도 좋은데,  타입이랑 필드명 모두 EntityStatus entityStatus 요렇게 가는 건 어떨까요?? 우선 각 도메인에 종속적인 상태는 아니고 모든 엔티티 객체이니 일관성을 줄 수 있다고 생각했습니다! 또한 이를 추상화 하기 위해서 Status라는 타입명으로 가게 된다면 자칫 어떤 Status인지를 식별하기 어려울 수도 있을 것 같아요!
```suggestion
    private Status favoriteStatus;
```

Reply:
넵 알겠습니다!